README.md

```diff
@@ -164,3 +165,39 @@ MyComponent.propTypes = {
   },
 }
 ```
+
+### `--prefer-type-aliases`
```

Comment:
I think I would prefer to use an option of `--declaration-style` with the default value being `interface`.

```suggestion
### `--declaration-style=type`
```

Reply:
Addressed